include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -201,9 +204,10 @@ struct is_temp_buff : ::std::false_type
 };
 
 template <typename _Iter>
-struct is_temp_buff<_Iter, ::std::enable_if_t<!is_hetero_it<_Iter>::value && !::std::is_pointer_v<_Iter> &&
-                                                  !is_passed_directly_it<_Iter>::value,
-                                              void>> : ::std::true_type
+struct is_temp_buff<
+    _Iter, ::std::enable_if_t<
+               !is_hetero_it_v<_Iter> && !::std::is_pointer_v<_Iter> && !is_passed_directly_it<_Iter>::value, void>>
```

Comment:
`is_passed_directly_it<T>::value` is only used twice, but could be another candidate for `_v` treatment

Reply:
```is_passed_directly_it``` has used inly in one place :
```C++
    template <typename _Iter>
    ::std::enable_if_t<is_passed_directly_it<_Iter>::value, __range_holder<oneapi::dpl::__ranges::guard_view<_Iter>>>
    operator()(_Iter __first, _Iter __last)
    {
        assert(__first < __last);
        return __range_holder<oneapi::dpl::__ranges::guard_view<_Iter>>{
            oneapi::dpl::__ranges::guard_view<_Iter>{__first, __last - __first}};
    }
```
So I think that creation of ```_v``` treatment is not required in this case.

Reply:
Fixed